### ISSUE TITLE
Stream the files (chunking) and start rendering  during the XHR request

### ISSUE THREAD

**Issue by bluejamesbond** [bluejamesbond/Scribe.js#53]

**Comment by bluejamesbond** [comment]
@guillaumewuip I am currently using scribe in a production env as an internal tool. This is a huge issue, so can you help me out with this?

**Comment by bluejamesbond** [comment]
@guillaumewuip Since there are numerous rows, is it possible to add row-by-row with angular without having angular to validate the previous rows? This way, we can add each row as they are streamed in.

**Comment by guillaumewuip** [comment]
yo @bluejamesbond. I'm quite busy these days and will be probably offline for the next 4 days. 
Anyway, I think is an angular `ng-repeat` "problem". There is simply to much things to render. 
Some guys have already done a lot of work somewhere in order to provide better `ng-repeat` behaviour with huge dataset.

**Comment by guillaumewuip** [comment]
Actually, I can't see how to "add" a row, without having angular to render the whole dataset

**Comment by guillaumewuip** [comment]
After googling a little bit, I found links I've already read : 
- http://kamilkp.github.io/angular-vs-repeat/#?tab=8
- http://stackoverflow.com/questions/17348058/how-to-improve-performance-of-ngrepeat-over-a-huge-dataset-angular-js
- http://ionicframework.com/docs/api/directive/collectionRepeat/

**Comment by bluejamesbond** [comment]
I'll go over it and see what I can so over this weekend. Might have some questions during the implementation.

**Comment by guillaumewuip** [comment]
No problem, I will check my emails :)

**Comment by Kyle-Falconer** [comment]
Yeah I'm running into this as well, even after just a couple minutes of running my program (which outputs a lot). The web console is pretty much unusable for me as it is right now, and my log file is about 20k lines and climbing.
